USING_MULTI_TENANCY.md

```diff
@@ -0,0 +1,53 @@
+# Multi-Tenancy
+
+Multi-tenancy allows a single `fhir-works-on-aws` stack to serve as multiple FHIR servers for different tenants.
+
+`fhir-works-on-aws` uses a pooled infrastructure model for multi-tenancy. This means that all tenants share the 
+same infrastructure (DynamoDB tables, S3 Buckets, Elasticsearch cluster, etc.), but the data 
+is logically partitioned to ensure that tenants are prevented from accessing another tenant’s resources.
+
+## Enabling multi-tenancy
+
+Use the `enableMultiTenancy` option when deploying the stack: 
+
+```bash
+serverless deploy --enableMultiTenancy true
+```
+
+**Note:** Updating an existing (single-tenant) stack to enable multi-tenancy is a breaking change. Multi-tenant 
+deployments use a different data partitioning strategy that renders the old, single-tenant, data inaccessible. 
+If you wish to switch from single-tenant to a multi-tenant model, it is recommended to create a new multi-tenant stack 
+and then migrate the data from the old stack. 
+
+## Tenant identifiers
+
+Tenants are identified by a tenant Id in the auth token. There are 2 ways to include a tenant Id in the auth token:
+
```

Comment:
Mention syntax the tenantId should follow?

Reply:
Done